src/amber/router/session/cookie_store.cr

```diff
@@ -0,0 +1,80 @@
+module Amber::Router::Session
+  # This is the default Cookie Store
+  class CookieStore < AbstractStore
+    property secret : String
+    property key : String
+    property expires : Int32
+    property store : Amber::Router::Cookies::Store
+    property session : Hash(String, String)
+
+    def initialize(@store, @key, @expires, @secret)
+      @session = current_session
+      @session[key] = SecureRandom.uuid
+    end
+
+    def id
+      @session[key]
+    end
+
+    def destroy
+      @session.clear
+    end
+
+    def [](key)
```

Comment:
I like symbols as well, in the case above since for cookie store the entire session is turned into a JSON when is it gets parse back to a hash it turns the keys into a string. Maybe there is a way in crystal to do this have string and symbol point to the same value. See also https://stackoverflow.com/questions/9712706/ruby-symbols-vs-strings-in-hashes